tests/integration/conftest.py

```diff
@@ -40,10 +40,11 @@ def app():
     app = create_app(
         DEBUG=True,
         WTF_CSRF_ENABLED=False,
-        CELERY_ALWAYS_EAGER=True,
+        CELERY_ALWAYS_EAGER=False,
```

Comment:
I don't understand: why importing locally isn't passing the correct celery app? Maybe it's too complex to answer.